.circleci/config.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       - run:
           name: Install dependencies
           command: |
-            pipenv sync --three --dev
+            pipenv install --skip-lock --three --dev
```

Comment:
Do we want to continue to build with `pipenv`?

Reply:
Yeah, for now.